CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+*                                   @chxchx @cjwagner @fejta @rkpagadala @yutongz
+/prow/config/                       @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid
+/prow/cluster/jobs/                 @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun
+/prow/cluster/jobs/istio/bots/      @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @ozevren
+/prow/cluster/jobs/istio/community/ @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun
+/prow/cluster/jobs/istio/installer/ @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @costinm
+/prow/cluster/jobs/istio/istio.io/  @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @frankbu
+/prow/cluster/jobs/istio/istio/     @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @costinm @rshriram
+/prow/cluster/jobs/istio/operator/  @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @ayj @costinm @Laci21 @martonsereg @matyix @ostromart @rcernich @waynz0r
+/prow/cluster/jobs/istio/pkg/       @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @ozevren
+/prow/cluster/jobs/istio/tools/     @chxchx @cjwagner @fejta @rkpagadala @yutongz @howardjohn @sdake @geeknoid @duderino @linsun @mandarjog
```

Comment:
Is this going to start requesting my review on every ProwJob config change?
I think oncall should have permission to approve changes to these files when needed, but most job config changes are outside the scope of oncall and are the responsibility of the job authors so oncall shouldn't be requested for review.

Reply:
Yes I do not want to get assigned as a reviewer on every PR in this repo. For the most part I never want to see anything in prow/cluster/jobs, want this delegated to the subject matter experts

Reply:
This is a problem we have on istio/istio as well. We want some people to be able to approve anything, but not actually be requested for review on every PR. I don't think we have a way to allow oncall type people to approve job changes without making them marked for review on every pr

Reply:
Use the better designed for large repos OWNERS files? :smiling_imp: